cmd/note_common.go

```diff
@@ -283,7 +283,9 @@ func createNote(rn string, isMR bool, idNum int, msgs []string, filename string,
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println(noteURL)
+	if !quickaction {
+		fmt.Println(noteURL)
+	}
```

Comment:
If a quick action is used with a comment the URL is still worthy showing, isn't it? It can be omitted only if no comments are given.

Reply:
The reason why I've added that is that in the case of _just_ using the quick action, the URL returned is `https://gitlab.com/lzampier/kernel-test/merge_requests/0#note_000000000`. 
It sure seems like a bug on GitLab's side, but since they treat quick actions as a different kind of `message`, I do understand they having nothing to point at.

Reply:
Hmm, got it. What if instead of then omitting the URL completely you check its suffix for the invalid `#note_000000000`?
I know it's not documented and might be a bug on their side and, possibly, get fixed one day. But if we get a test at least for this case we can make sure we it failing as soon as the behavior changes on their side.

I just really like the URLs left in the stdout after the user performs an action that effectively created a valid URL he can click on :). But feel free to disagree.

Reply:
Wait, first things first, the URL is __not__ shown only when the quick action is sent only thing sent, like:
```shell
$ lab mr approve 
Merge Request !1 approved
```
If there is a comment, the output is:
```shell
$ lab mr approve -m "Im a comment"
https://gitlab.com/lzampier/kernel-test/merge_requests/1#note_683541416
Merge Request !1 approved
```
Knowing that, do you still want me to show that  '000*' URL? btw I'm all in for that since I can pretty much drop that `quickaction bool` var, and make the createNote function a bit cleaner :D

Reply:
Ok, I miss understood part of the code! That's my fault.
The `quickaction` variable name in `createNote()` mislead me because it's only `true` when the user wants to also comment (`note == true`). So my thought was: "wait, but the approval without comments are also being done with quick actions, so the `quickactions` var will be `true`there as well."

Maybe changing the name of the variable or adding some comments there would be helpful to avoid this kind of back and forth to understand what's going on, not a hard requirement, just IMO :), your call. But yes, the code is correct!

Reply:
Yeah, thank you for pointing that out, and I agree the code is not particularly clear on that point, I will submit a follow-up patch.